### PR TITLE
Revamped the Best Tweets Page

### LINKED ISSUE #1969: Redesign the Best tweets project
**Describe the Feature you want to end**
I want to add hover effect on the tweets and also some UI changes

**Screenshots (optional)**
Right now it looks blunt
image

### PR DESCRIPTION
**Pull request by PoulavBhowmick03**
# Title and Issue number 
Redesign the Best tweets project #1969

Close #1969 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`

image

Also added hover effect

### CODE REVIEW

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121

**Comment by PoulavBhowmick03** [comment]
@pranjay-poddar @MohitGupta121 can you pls merge this? I want to make other PRs

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121 Have a loot at it

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "liveServer.settings.port": 5501
+    "liveServer.settings.port": 5502
```
Comment (MohitGupta121) [inline .vscode/settings.json:2]: please not change this port

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@PoulavBhowmick03 PTAL, Thanks.

**Review by PoulavBhowmick03 — COMMENTED** [review]
Changed it

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121 Made the changes

**Review by MohitGupta121 — APPROVED** [review]
Thanks @PoulavBhowmick03 LGTM.

File: .vscode/settings.json (hunk 1 above)
Comment (MohitGupta121) [reply .vscode/settings.json]: Done.

**Pull request merged**